ecs/security-groups.tf

```diff
@@ -62,7 +62,7 @@ resource "aws_security_group" "ecs" {
 resource "aws_security_group_rule" "ecs" {
   for_each                 = toset(var.allow_internal_traffic_to_ports)
   type                     = "ingress"
-  from_port                = 0
```

Comment:
I don't quite understand about this security group rule, so not really sure whether this changes is valid.

Reply:
we open up the port for the web app by default. If we want to use any other port for some service that is not our web server, e.g. a pdf-renderer running in its own service, then we need to allow traffic for this port.

E.g. in `allow_internal_traffic_to_ports` we would pass `[ 5000 ]` to allow traffic for port 5000. In the current setting, we would now open up all ports from 0 - 5000. This was never intended.